staging/src/k8s.io/apiserver/pkg/cel/environment/base.go

```diff
@@ -139,6 +141,14 @@ var baseOpts = []VersionedOptions{
 	},
 }
 
+var costOption = VersionedOptions{
+	// This is to configure the cost calculation for extended libraries
+	IntroducedVersion: version.MajorMinor(1, 0),
+	ProgramOptions: []cel.ProgramOption{
+		cel.CostTracking(&library.CostEstimator{}),
+	},
+}
+
```

Comment:
I thought we are gonna enable the feature gate in the current release then the future usage would get `strictCostOps` since 1.31. Currently the feature gate is only used under baseEnv so it will be available across k8s as long as the cel compiled through baseEnv

Reply:
Sorry for the confusion. Updated to only affected GA features.